src/cmdlib.sh

```diff
@@ -1038,3 +1038,12 @@ buildmeta = builds.get_build_meta('${buildid}')
 cmdlib.import_ostree_commit(workdir, builddir, buildmeta)
 ")
 }
+
+# Extract the value of NAME from os-release
+extract_osrelease_name() {
+    local buildid=$1; shift
+    local out="$workdir/tmp/osrelease"
+    rm "${out}" -rf
+    ostree checkout --repo "${tmprepo}" --user-mode --subpath=/usr/lib/os-release "${buildid}" "$out"
+    . "$out/os-release" && echo "${NAME}"
```

Comment:
Won't this pollute the local namespace? Can we wrap it in parentheses? Also I think we'll need a `# shellcheck disable=SC2153` above.